.travis.yml

```diff
@@ -23,6 +23,7 @@ branches:
     - integration
     - staging
     - prod
+    - issues/jessebrennan/556-chalice-terraform
```

Comment:
Yes. If you look at the commit history it should be clear that this is a temporary change that will be removed before this PR is merged. It's a result of this PR being "chained" onto #556. It will need to be rebased once #556 is merged, at which point I will remove the commit containing this change.

Reply:
thx!